src/assets/styles/core/media.scss

```diff
@@ -0,0 +1,13 @@
+img {
+  max-width: 100%;
+}
+
+figure {
+  margin: 0;
+}
+
+img,
+picture {
+  margin: 0;
+  max-width: 100%;
```

Comment:
I see `img` declaration above it that also has `max-width` property. Can we remove the one above?